### PR TITLE
Removes air alarm access from engineers

### PR DESCRIPTION
**Pull request by S34NW**
## What Does This PR Do
This PR makes it so only people with atmos access can access air alarm settings.

## Why It's Good For The Game
Atmos has been less relevant as of late, with no wall-melting plasmafires and the SM being a gas-based engine that atmos for some reason has no access to.
This PR would mean engineers have to work in tandem with atmos (or just cut the wires of the alarm, I guess) to repressurise a breach or fix something, as opposed to just doing it all themselves with their lathe-printed RPD.
Atmos techs cant mess with APCs, why should the reverse be true?

## Changelog
:cl:
tweak: Engineers no longer have air alarm access
/:cl:

### CODE REVIEW

**Comment by FlattestGuitar** [comment]
The problem with atmos is they're useless and they're a role that is only around because it sounds theoretically cool, but in practice it's nigh impossible to do something interesting with it. I'd personally consider merging with engineers to save people the trouble.

On top of it all, you have the duality of both roles - you pick them because you want to RP or build something cool, but each shift there's a 50/50 chance you'll actually have to fix something, at which point you're taken out of your RP or building to patch holes, something that's incredibly dull and devoid of meaningful RP opportunities.

This PR just makes it harder for people to give a damn when they already barely do.

**Comment by Wojz** [comment]
It sucks to show up to a construction site to learn that engineers have already done my job. I'm all for this.

**Review by Kyep — APPROVED** [review]

**Comment by AffectedArc07** [comment]
I would rather engineers be unable to get RPDs over this. Some things an engineer should be able to do (like reset an alarm in an area), whereas atmos focuses on the actual atmospherics infrastructure itself (The pipes, atmos room itself, etc etc)

**Comment by S34NW** [comment]
> 
> 
> I would rather engineers be unable to get RPDs over this. Some things an engineer should be able to do (like reset an alarm in an area), whereas atmos focuses on the actual atmospherics infrastructure itself (The pipes, atmos room itself, etc etc)
> 
> TLDR: Not in favour

That would be nice, but I don't see a good way to do this apart from making an RPD a "unique" item like atmos holofan and removing it from lathes, but that creates its own set of issues for civvie build projects. Do you have any general ideas on how you'd like to see something like that implemented?

**Comment by AffectedArc07** [comment]
> That would be nice, but I don't see a good way to do this apart from making an RPD a "unique" item like atmos holofan and removing it from lathes, but that creates its own set of issues for civvie build projects. Do you have any general ideas on how you'd like to see something like that implemented?

I would honestly be fine with taking the RPD out of autolathes. Any random civvie or cargo tech shouldnt have the power to manipulate the pipenet and disposals, atmos should have a use in the round, with cargo asking for custom disposals or whatever.


This PR also has 2 maints against, so will be closed.
